documentation/source/swagger/parts/schemas/invoice_financings_apis/InvoiceFinancingsContract.yaml

```
@@ -41,86 +34,79 @@ properties:
         3. Para os demais clientes (tipos de cliente 3, 4, 5 e 6), informar 14 (catorze) posições com complemento de zeros à esquerda se a identificação tiver tamanho inferior;
       - **Código do contrato:** 1 (uma) até 40 (quarenta) posições, sem complemento de caracteres.
     example: '92792126019929279212650822221989319252576'
-  requestDateTime:
-    type: string
-    maxLength: 20
-    format: DateTimeString
-    pattern: '^(\d{4})-(1[0-2]|0?[1-9])-(3[01]|[12][0-9]|0?[1-9])T(?:[01]\d|2[0123]):(?:[012345]\d):(?:[012345]\d)Z$'
-    example: "2020-01-05T08:30:00Z"
-    description: Data e hora da consulta, conforme especificação RFC-3339, formato UTC.
   productName:
     type: string
     maxLength: 140
     pattern: \w*\W*
     description: |
       Denominação/Identificação do nome da Modalidade da Operação de Crédito divulgado ao cliente
-    example: "Desconto de Cheques"
+    example: AD
```

Deveria ser uma string, certo ?!
```suggestion
    example: "AD"
```